packages/cdk8s-cli/bin/cmds/import.ts

```diff
@@ -59,7 +60,7 @@ function parseImports(file: string): ImportSpec {
   // like this: url.com/crds?crd=mycrd.yaml
   console.error('If you reach here, please open an issue with cdk8s.');
```

Comment:
Throw an exception: "unable to parse import specification. syntax is [NAME=]SOURCE